lib/lang/latin/latin-language-dataset.js

```diff
@@ -368,8 +384,29 @@ export default class LatinLanguageDataset extends LanguageDataset {
     return this
   }
 
+  getVerbsIrregularLemma (verbForm) {
+    let res = this.pos.get(Constants.POFS_VERB).types.get(Form).items.map(item => { return { val: item.value, lemma: item.features[Feature.types.hdwd].value, fullForm: item.features[Feature.types.fullForm].value } })
+    let res1 = {}
```

Comment:
Can we rename `res` and `res1` so that they will reflect a meaning of data they contain?